web/index.html

```diff
@@ -94,7 +102,7 @@
           <div class="inner"></div>
         </div>
         <div class="view" flex two layout vertical>
-          <div class="header" layout horizontal wrap>
+          <div class="header small" layout horizontal wrap>
             <li><a id="consoletab" selected><span>Console</span></a></li>
```

Comment:
I'm trying to make that tab area appear secondary in comparison to the main dart/html/css one. I'll dial the font size up slightly.